tubular/base.py

```diff
@@ -89,10 +98,6 @@ def __init__(
             msg = f"{self.classname()}: columns must be a string or list with the columns to be pre-processed (if specified)"
             raise TypeError(msg)
 
-        if not isinstance(copy, bool):
-            msg = f"{self.classname()}: copy must be a bool"
-            raise TypeError(msg)
-
         self.copy = copy
```

Comment:
I've kept it in as otherwise tests fails as transformers expect there to be a copy attribute as it's an argument in transformer init methods. Until we fully deprecate this method I don't think we're able to drop this line